cli/src/main/java/com/walmartlabs/concord/cli/Run.java

```diff
@@ -256,10 +270,11 @@ public Integer call() throws Exception {
                 new CliServicesModule(secretStoreDir, targetDir, defaultTaskVars, new VaultProvider(vaultDir, vaultId), dependencyManager, verbosity))
                 .create();
 
+        // TODO ask brig why [separate loader], why [same targetDir], why [no imports]
```

Comment:
At this point, the entire project is already loaded, all imports have been processed, and everything is placed in the `targetDir`. this load will handle all files, including the imported ones...

when we load the project for the first time, we don't yet have an Injector, no listeners will be triggered :(

I think these listeners aren’t actually used anywhere—probably best to just get rid of them:)

Reply:
Agree https://github.com/walmartlabs/concord/pull/1015